README.md

```diff
@@ -22,6 +22,8 @@ Es ist aufwendig für Lerngruppen oder Kurse Dateien miteinander zu teilen, wenn
 
 ## Dokumentation
 
-Hier kann die Dokumentation gefunden werden: [Dokumentation](/doc/documentation.adoc)
+Hier kann die Dokumentation gefunden werden: [Dokumentation](/doc/html5/documentation.html)
```

Comment:
use the github pages link here

Reply:
that should've been there. I think something didn't got pushed.